pax/simulation.py

```diff
@@ -104,6 +104,35 @@ def __init__(self, config_to_init):
         else:
             self.s2_patterns = None
 
+        ##
+        # Load pdf for single photoelectron, if available
+        ##
+        if c.get('photon_area_distribution'):
+            # Extract the spe pdf from a csv file in form of bin, prob.
+            charge = []
+            p = []
+            with open(utils.data_file_name(c['photon_area_distribution']), "r") as f:
+                for line in f.readlines():
+                    q = float(line.split(",")[0])
+                    prob = float(line.split(",")[1])
+                    # don't allow negative charge, which can occur from noise subtraction
+                    prob = prob if q >= 0 else 0
+                    charge.append(q)
+                    p.append(prob)
+
+            # Create a converter from uniform random numbers to SPE gains
+            # note not really pdf, does not necessarily sum to 1 due to forcing probability of negative charge to be 0
+            spe_bins, spe_pdf = np.array(charge), np.array(p)
+            # change spe_bins to have mean equal to 1
+            mean_spe = (spe_bins * spe_pdf).sum() / spe_pdf.sum()
+            spe_bins = spe_bins/mean_spe
+            #convert pdf -> cdf
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->